content/posts/0.22-release.md

```diff
@@ -155,6 +155,34 @@ let f = o.py_get_attr.bind(o);
 f("a"); // 3
 ```
 
+## The new Pyodide CLI interface
```

Comment:
My thought is to organize the sections roughly in order of increasing length, so maybe put this before the longer ffi section?

Reply:
> in order of increasing length

I would normally write in order of importance (things we want to emphasize), but in order of increasing length sounds also nice to me.

Reply:
+1 also for the order of importance. Or I would say let's maybe just merge as is